gui/public/i18n/en/translation.ftl

```diff
@@ -469,6 +469,8 @@ settings-serial-get_infos = Get Infos
 settings-serial-serial_select = Select a serial port
 settings-serial-auto_dropdown_item = Auto
 settings-serial-get_wifi_scan = Get WiFi Scan
+settings-serial-file_type = Serial log
```

Comment:
Is this really correct? Why not "Text file" or "Plain text"?

Reply:
i mean sure? idk lol